### PR TITLE
Added pretty-bytes

### LINKED ISSUE #2: Improve size output in CLI
Currently, it's shown in bytes. We could use https://github.com/sindresorhus/pretty-bytes to make it look better.

### PR DESCRIPTION
**Pull request by hafffe**
Fixes #2

### CODE REVIEW

**Comment by gillstrom** [comment]
Thanks! :-)))))))))))))

**Comment by kevva** [comment]
Hmm, I don't think we should use `pretty-bytes` in the API though, just the CLI. If someone else want to display the size in another way it'll be incredibly hard to parse now.

**Comment by gillstrom** [comment]
Whops.. yepp.. I'll change the commit!

**Comment by hafffe** [comment]
Yeah, I should thought of that!
